### ISSUE TITLE
the *nix install script is not found

### ISSUE THREAD

**Issue by trunghlt** [spf13/spf13-vim#640]
the *nix install script is not found

**Comment by ghost** [comment]
can you tell us what you tried to do and what didn't work?

**Comment by trunghlt** [comment]
I tried to download https://j.mp/spf13-vim3 but it is not found.

**Comment by ghost** [comment]
I'm able to click that link and see the file.   Is it a 404 error for you or something else?

If you want to download it immediately , you can grab https://raw.githubusercontent.com/spf13/spf13-vim/3.0/bootstrap.sh and run it.

**Comment by ljani** [comment]
Possible duplicate: #615

**Comment by trunghlt** [comment]
thank jrobeson, yes it's 404.

**Comment by aVolpe** [comment]
You can try https://cdn.rawgit.com/spf13/spf13-vim/3.0/bootstrap.sh (I have the same problem with the raw.githubuser file).

**Comment by ghost** [comment]
@aVolpe : very strange as the githubusercontent url is the same as if you were to browse the repositories via github.com and then click the raw button on a specific file. Perhaps you should contact github?

**Comment by aVolpe** [comment]
Yes, If I navigate the repository I can see the file, but the button raw throw the 404. I think is a very strange problem because throw a proxy it's visible (i.e. anonymouse).

**Comment by ghost** [comment]
it's not a problem that we can fix. you must talk to github.

**Comment by ghost** [comment]
@spf13 : more than 1 person seems to be affected by this. any ideas? maybe host on your own server via another shorturl?

**Comment by aVolpe** [comment]
@jrobeson  I know, I only want to add a workaround for @trunghlt!, I am using spf13-vim since last year without this problem.

**Comment by aVolpe** [comment]
@jrobeson  the rawgit link is permanent and I think it is reliable.

**Comment by ghost** [comment]
oh i thought you said it 404ed too

**Comment by aVolpe** [comment]
@jrobeson  the `cdn.rawgit.com` works fine, the `raw.github.com` doesn't work sometimes.

**Comment by spf13** [comment]
Sounds like the solution is just to change the j.mp link to point to the cdn.rawgit.com version. I can do that.

**Comment by ghost** [comment]
ping @spf13

**Comment by spf13** [comment]
Trying to update the j.mp destination. Doesn't appear to be possible. Will have to create a new one and change all the installation documentation.

**Comment by 0x64746b** [comment]
FTR: http://j.mp doesn't resolve for me at all. Maybe you want to update at least the link at http://vim.spf13.com/? Thx.

**Comment by ghost** [comment]
@spf13: did you change that yet?
